.github/workflows/zxc-compile-pbj-code.yaml

```diff
@@ -139,12 +139,7 @@ jobs:
           time_unit: seconds
           junit_files: "${{ env.PBJ_INTEGRATION_TESTS }}/**/build/test-results/**/*.xml"
 
-      - name: Gradle JMH Jars (PBJ Integration)
-        if: ${{ inputs.enable-jmh-tests && steps.gradle-integration-build.conclusion == 'success' && !cancelled() && always() }}
-        working-directory: ${{ env.PBJ_INTEGRATION_TESTS }}
-        run: ./gradlew jmhJar
```

Comment:
`jmhJar` is build automatically as part of the next step. I propose to remove this step here to simplify the GH action setup.